docs/tutorial/hyperfoil.adoc

```diff
@@ -1,129 +1,156 @@
-      - sh: "curl -Ls https://sh.jbang.dev | bash -s - trust add https://repo1.maven.org/maven2/io/quarkus/quarkus-cli/"
-      - sh: "curl -Ls https://sh.jbang.dev | bash -s - app install --fresh --force quarkus@quarkusio"
+  remove-hyperfoil:
+  - sh: cd /tmp && rm -fR hyperfoil*
 
-  getting-started:
+  setup-getting-started:
   - sh: cd /tmp/
   - sh: "[[ ! -d quarkus-quickstarts ]] && git clone https://github.com/quarkusio/quarkus-quickstarts.git"
   - sh: cd quarkus-quickstarts
   - sh: cd getting-started
   - script: ensure-quarkus-cli
+  - sh: export GETTING_STARTED_DIR="$(pwd)" #set an environment variable
+
+  ensure-quarkus-cli:
+  - sh: quarkus version
+    then:
+    - regex: "command not found"
+      then:
+      - sh: "curl -Ls https://sh.jbang.dev | bash -s - trust add https://repo1.maven.org/maven2/io/quarkus/quarkus-cli/"
+      - sh: "curl -Ls https://sh.jbang.dev | bash -s - app install --fresh --force quarkus@quarkusio"
+      - sh: "source ~/.bashrc"
+
+  run-getting-started:
+  - sh: cd $GETTING_STARTED_DIR #ensure we are in the correct directory
   - sh:
       command: quarkus dev
       prompt:
-        "Press [r] to resume testing, [o] Toggle test output, [:] for the terminal, [h] for more options>": "r"
+        "Press [e] to edit command line args (currently ''), [r] to resume testing, [o] Toggle test output, [:] for the terminal, [h] for more options>": "r"
     watch:
-    - regex: "Tests paused"
+    - regex: "Tests completed"
       then:
       - signal: ready
     on-signal:
       done:
-      - ctrlC #exits the process
-
+      - ctrlC
 hosts:
-  test: me@localhost #replace this with your ssh username@hostname
+  test: ${{target-host}}
 roles:
-  example:
+  setup:
     hosts:
     - test
+    setup-scripts:
+    - setup-getting-started
+    run-scripts:
+    - run-getting-started
+  hyperfoil:
+    hosts:
+    - test
+    setup-scripts:
+    - setup-hyperfoil
     run-scripts:
     - test-endpoint
-    - getting-started
+    cleanup-scripts:
+    - remove-hyperfoil
+EOF
+```
+
+Run with and replace `targethost` value with your needs
+```
+targethost=$USER@localhost
+java -jar qDup-0.8.5-uber.jar -i ~/.ssh/qdup hyperfoil.yaml -ix -S target-host=$targethost
 ```
 
 == Splitting the roles
-The previous tutorial created an `example` role to run all of ours scripts. This works but a proper performance test will normally isolate the load generation (currently `curl`) from the process we are testing (currently `quarkus dev`).
-qDup isolates scripts by assigning them to roles that are running on different `hosts`. To isolate `test-endoing` from `getting-started` we need to introduce a second role. Let us call it `hyperfoil`. The `roles` section will now look like the following:
+The previous tutorial created a role to run all of ours scripts. This works but a proper performance test will normally isolate the load generator (`hyperfoil`) from the process we are testing (`quarkus dev`).
```

Comment:
(e.g. `hyperfoil`)

Reply:
Removed `(`hyperfoil`)` and `(`quarkus dev`)`